akka-http-core/src/main/scala/akka/http/impl/util/LogByteStringTools.scala

```diff
@@ -17,16 +17,25 @@ private[akka] object LogByteStringTools {
   def logByteStringBidi(name: String, maxBytes: Int = MaxBytesPrinted): BidiFlow[ByteString, ByteString, ByteString, ByteString, NotUsed] =
     BidiFlow.fromFlows(
       logByteString(s"$name DOWN", maxBytes),
-      logByteString(s"$name UP", maxBytes)
+      logByteString(s"$name UP  ", maxBytes)
+    )
+
+  def logToStringBidi[A](name: String, maxBytes: Int = MaxBytesPrinted): BidiFlow[A, A, A, A, NotUsed] =
+    BidiFlow.fromFlows(
+      logToString(s"$name DOWN", maxBytes),
+      logToString(s"$name UP  ", maxBytes)
```

Comment:
useful when debugging what event is flying where, would keep those utils if you don't mind